mysite/missions/templates/missions/pipvirtualenv/removing_packages.html

```diff
@@ -0,0 +1,107 @@
+{% extends 'missions/pipvirtualenv/base.html' %}
+{% comment %}
+# This file is part of OpenHatch.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+{% endcomment %}
+
+{% block mission_main %}
+<div class='submodule fat'>
+  <div class='head'>
+    <h3>Removing packages and deactivating virtualenvs</h3>
+  </div>
+  <div class="body">
+
+  <h3>Deactivating virtualenvs</h3>
+  <p>To deactivate a virtualenv, simply type <kbd>deactivate</kbd> at the
+  terminal and press Enter.</p>
+  
+  <p>You'll notice that after running the <kbd>deactivate</kbd> command,
+  your terminal prompt (something like <kbd>C:\Users\username></kbd> on
+  Windows or <kbd>user@hostname:~ $</kbd> on Linux/Mac OS X) will no longer
+  be prefixed by the name of a virtualenv. This means that no virtualenv is
+  active at the moment.</p>
+
+  <p>To activate a virtualenv, at the terminal, type <kbd>workon</kbd>,
```

Comment:
Using virtualenvwrapper, workon will activate a virtualenv. Otherwise with no virtualenvwrapper, it is source TestEnv/bin/activate.

I'm not sure the best way to keep this simple. I should not that I did not run Glyph's script on my system since I didn't wish for it to change my configuration on OS X.